server/routes/index.ts

```diff
@@ -27,6 +33,13 @@ export default function Index({ userService, prisonerService, licenceService, co
     get('/prisoner/:nomsId/image', otherAccessRoutes.getPrisonerImage)
   }
 
+  // Routes for creating a licence
+  const licenceRoutes = () => {
+    get('/licence/staffId/:staffId/caseload', licenceAccessRoutes.getUserCaseload)
```

Comment:
We are going to end up having a lot of routes, and a lot of route handlers inside one class. I was thinking of splitting those out into classes which just have one GET and one POST method (if applicable).. just a thought

Reply:
In my mind I was thinking that we would group the related routes for one `workflow` into the same class. So maybe `licenceRoutes` is not the right classification - might be better as `createLicenceRoutes` and `varyLicenceRoutes` etc..

Reply:
^ the comment in the code about is right, but the name isn't :-)